src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -926,24 +934,24 @@ PaError PaPulseAudio_StartStreamCb( PaStream * s )
         goto startstreamcb_error;
     }
 
-    /* Make sure we pass no error on intialize */
+    /* It must be ensured that no errors are encountered during
+     * initialization.
+     */
```

Comment:
This comment (and the original version) is confusing. I think better would be:

```
    /* Success! */
```

Reply:
Removed altogether as it didn't add nothing to comments.